README.md

```diff
@@ -1 +1 @@
-# project-calculator
\ No newline at end of file
+## See a live demo [Here](https://josia-calculator.herokuapp.com/
```

Comment:
dont forget to edit more on the README regarding general things like author/contributor, technologies, setup commands before you merge etc